frontend/src/global_styles/fonts/_openproject_icon_definitions.scss

```diff
@@ -1,1536 +1,1548 @@
 @mixin icon-mixin-accessibility {
-    content: "\f101";
+    content: '\f101';
```

Comment:
code climate wanted me to change just two of these definitions but had all these quotes in their internal list of "known bad formatting errors". So I fixed them all instead of just my two quotes. 🤷